docs/dapps/smart-contracts/using-foundry-with-the-avalanche-c-chain.md

```diff
@@ -0,0 +1,179 @@
+# Using Foundry with the Avalanche C-Chain
+
+## Introduction
+
+This article shows how to deploy and interact with smart contracts using foundry on the [Fuji C-Chain](../../quickstart/fuji-workflow.md).
+
+[Foundry](https://github.com/foundry-rs/foundry) is a smart contract development toolchain written in Rust.
+
+Foundry manages your dependencies, compiles your project, runs tests, deploys, and lets you interact with the chain from the command-line.
+
+## Prerequisites
+
+- You have installed [Foundry](https://github.com/foundry-rs/foundry#installation); This installation includes the `forge` and `cast` binaries used in this walk-through
```

Comment:
1. Please make the link be for `installed Foundry` and not just for `Foundry`.

```text
installed [Foundry](https://github.com/foundry-rs/foundry#installation); 
```


Should be 

```text
[installed Foundry](https://github.com/foundry-rs/foundry#installation); 
```

2. Replace the `;` at the end of the sentence with a `.`